.github/linters/.eslintrc.json

```diff
@@ -225,21 +225,6 @@
                 "react/no-unused-prop-types": "off"
             }
         },
-        {
```

Comment:
#### Comment, no changes needed: 
This was probably a bad approach to begin with. Better would have been to ignore the line or the entire file rather than adding a new global rule for one file.